src/binary/ibc-setup/commands/connect.ts

```diff
@@ -80,7 +94,13 @@ export async function run(options: Options, app: AppConfig, logger: Logger) {
 
   const nodeA = await signingClient(srcChain, options.mnemonic);
   const nodeB = await signingClient(destChain, options.mnemonic);
-  const link = await Link.createWithNewConnections(nodeA, nodeB);
+  const link = await Link.createWithNewConnections(
+    nodeA,
+    nodeB,
+    logger,
+    options.srcTrust ?? undefined,
+    options.destTrust ?? undefined
```

Comment:
Following our convention (`undefined` for not defined variables and `null` for values intentionally set to nothing) `srcTrust` and `destTrust` resolve to `number | null` but `createWithNewConnections` Link function accepts `number | undefined`.

Perhaps we should change the definition of `createWithNewConnections` to accept `null` too?

Reply:
Sure, make it accept `number | undefined | null` if you prefer that. And same with the functions it calls into

Reply:
e4b7091